crates/crypto/merkle/src/lib.rs

```diff
@@ -145,25 +135,25 @@ mod tests {
         assert!(is_valid_normalized_merkle_branch(
             leaves[0],
             &proof_0,
-            2 * depth,
+            1 << depth,
             root
         ));
         assert!(is_valid_normalized_merkle_branch(
             leaves[1],
             &proof_1,
-            2 * depth + 1,
+            1 + (1 << depth),
             root
         ));
         assert!(is_valid_normalized_merkle_branch(
             leaves[2],
             &proof_2,
-            2 * depth + 2,
+            2 + (1 << depth),
             root
         ));
         assert!(is_valid_normalized_merkle_branch(
             leaves[3],
             &proof_3,
-            2 * depth + 3,
+            3 + (1 << depth),
```

Comment:
These are tests so I think readability is nice, I don't think there really was a reason to do change this right?

Reply:
Sounds good, what do you think of moving the addition to the other side then? so `(1 << depth) + 3` I find it a little easier to read, up to you though it isn't the biggest deal.